### PR TITLE
Basically redid the Russian roulette command

### PR DESCRIPTION
**Pull request by RopraMMC**
I re did the cmd without using if and else and clutter

### CODE REVIEW

File: src/cogs/fun.py [hunk 1]
```diff
@@ -188,25 +188,27 @@ async def casino(self, ctx: Context) -> None:
...
+                      colour=embed_colour[random_choice]).set_footer(
+            text=f"{ctx.author.name} {random_footer}",
             icon_url=(ctx.author.display_avatar.url)
         )
+
+
         return await ctx.send(embed=embed)
 
 
+
```
Comment (Pandabweer) [inline src/cogs/fun.py:212]: A space too many here...

File: src/cogs/fun.py [hunk 2]
```diff
@@ -188,25 +188,27 @@ async def casino(self, ctx: Context) -> None:
...
+        embed_colour = {"🌹 / **You lived**":0x32CD32, "<:gun:931861130488467456> / **You died**":0x8B0000}
+        ##randomly chooses the footer text
+        random_footer = random.choice(["loves to play this game", "must like excitement", "is definitely a risk taker",
+                                       "definitely hates life", "plays this game 24/7", "has issues"
+                                       "probably needs some help"])
+        ##making the embed
+        embed = Embed(description=f"{random_choice}",
+                      colour=embed_colour[random_choice]).set_footer(
+            text=f"{ctx.author.name} {random_footer}",
             icon_url=(ctx.author.display_avatar.url)
```
Comment (Pandabweer) [inline src/cogs/fun.py:205]: Unnecessary parenthesis

File: src/cogs/fun.py [hunk 3]
```diff
@@ -188,25 +188,27 @@ async def casino(self, ctx: Context) -> None:
...
+        ##makes the description of the embed
+        random_choice = random.choice(['🌹 / **You lived**','<:gun:931861130488467456> / **You died**'])
+        ##chooses the color of the embed
+        embed_colour = {"🌹 / **You lived**":0x32CD32, "<:gun:931861130488467456> / **You died**":0x8B0000}
+        ##randomly chooses the footer text
+        random_footer = random.choice(["loves to play this game", "must like excitement", "is definitely a risk taker",
+                                       "definitely hates life", "plays this game 24/7", "has issues"
+                                       "probably needs some help"])
+        ##making the embed
+        embed = Embed(description=f"{random_choice}",
```
Comment (Pandabweer) [inline src/cogs/fun.py:202]: This spacing and newlines is really inconsistent.

File: src/cogs/fun.py [hunk 4]
```diff
@@ -188,25 +188,27 @@ async def casino(self, ctx: Context) -> None:
...
-                icon_url=(ctx.author.display_avatar.url)
-            )
-            return await ctx.send(embed=embed)
-
-        embed = Embed(description="🌹 / **You lived**",
-                       colour=0x32CD32).set_footer(
-            text=f"{ctx.author} survived.",
+    @command(aliases=["rr", "gun_game", "russianroulette", "gungame"])
+    async def russian_roulette(self, ctx: Context):
+        ##makes the description of the embed
```
Comment (Pandabweer) [inline src/cogs/fun.py:193]: Comments are only one, example:
# this is a valid pep8 comment

**Review by Pandabweer — CHANGES_REQUESTED** [review]
I want changes look at the comments for more info.

**Comment by RopraMMC** [comment]
Ok doing changes now

File: src/cogs/fun.py [hunk 5]
```diff
@@ -188,21 +188,16 @@ async def casino(self, ctx: Context) -> None:
...
+    @command(aliases=["rr", "gun_game", "russianroulette", "gungame"])
+    async def russian_roulette(self, ctx: Context):
+        random_choice = random.choice(['🌹 / **You lived**','<:gun:931861130488467456> / **You died**'])
+        embed_colour = {"🌹 / **You lived**":0x32CD32, "<:gun:931861130488467456> / **You died**":0x8B0000}
+        random_footer = random.choice(["loves to play this game", "must like excitement", "is definitely a risk taker",
+                                       "definitely hates life", "plays this game 24/7", "has issues"
+                                       "probably needs some help"])
+        embed = Embed(description=random_choice,
+                      colour=embed_colour[random_choice]).set_footer(
+            text=ctx.author.name + " " + random_footer,
```
Comment (Pandabweer) [inline src/cogs/fun.py:200]: We use f-strings.

File: src/cogs/fun.py [hunk 6]
```diff
@@ -188,21 +188,16 @@ async def casino(self, ctx: Context) -> None:
...
-                       colour=0x32CD32).set_footer(
-            text=f"{ctx.author} survived.",
+    @command(aliases=["rr", "gun_game", "russianroulette", "gungame"])
+    async def russian_roulette(self, ctx: Context):
+        random_choice = random.choice(['🌹 / **You lived**','<:gun:931861130488467456> / **You died**'])
+        embed_colour = {"🌹 / **You lived**":0x32CD32, "<:gun:931861130488467456> / **You died**":0x8B0000}
+        random_footer = random.choice(["loves to play this game", "must like excitement", "is definitely a risk taker",
+                                       "definitely hates life", "plays this game 24/7", "has issues"
+                                       "probably needs some help"])
+        embed = Embed(description=random_choice,
```
Comment (Pandabweer) [inline src/cogs/fun.py:198]: Still inconsistent.

**Review by Pandabweer — CHANGES_REQUESTED** [review]

File: src/cogs/fun.py (hunk 6 above)
Comment (Pandabweer) [reply src/cogs/fun.py:198]: The embed part

**Review by Pandabweer — APPROVED** [review]

**Review by CaedenPH — APPROVED** [review]

**Pull request merged**